CONTRIBUTING.md

```diff
@@ -224,13 +224,11 @@ For all resources, if the environmental variable is not provided, a new instance
 
 ##### IoT Central
 
-IoT Central integration tests can be run against a specific available api version using command line argument "--api-version"
+`pytest azext_iot/tests/central/ -k "_int.py"`
```

Comment:
Is this section still needed? Id assume that readers would be able to infer how to run central tests from line 128

Reply:
You are right. 😹

Reply:
Removed.